motor_info:
  min_pwm: 0.6 # 1200 [ms]
  max_pwm: 0.975 # 1950 [ms]
  min_thrust: 0 #  [N]
  force_landing_thrust: 1.1 #[N]
  m_f_rate: -0.011
  pwm_conversion_mode: 0 # SQRT Mode
  vel_ref_num: 4
  ref1:
    voltage: 16
    max_thrust: 12 # N
    polynominal2: 0.027952492 # 0.0027952492 # x10
    polynominal1: -1.450135122 # -0.1450135122 # x10
    polynominal0: 0.7839164615 # 0.7839164615 # x1
